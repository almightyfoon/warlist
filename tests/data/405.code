c1501b_-0n8U9cc09Aakanaj9be59Cbz___-0nbSeO9c9G9yakanaj9ce59cbzbz