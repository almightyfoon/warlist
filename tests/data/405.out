https://conflictchamber.com/#c1501b_-0n8U9cc09Aakanaj9be59Cbz___-0nbSeO9c9G9yakanaj9ce59cbzbz

Cygnar Army - 79 / 75 points
[Theme] Heavy Metal

!!! This army must contain 75 points or fewer

[Haley 2] Major Victoria Haley [+25]
 - Firefly [8]
 - Stormwall [39]
 - Thorn [13]
 - Squire [0(5)]
Captain Arlan Strangewayes [0(4)]
Journeyman Warcaster [0(4)]
 - Charger [9]
Lieutenant Allison Jakes [4]
 - Cyclone [13]
Storm Strider [18]

Cygnar Army - 79 / 75 points
[Theme] Heavy Metal

!!! This army must contain 75 points or fewer

[Nemo 3] Artificer General Nemo [+25]
 - Dynamo [18]
 - Firefly [8]
 - Ironclad [12]
 - Lancer [10]
 - Squire [0(5)]
Captain Arlan Strangewayes [0(4)]
Journeyman Warcaster [0(4)]
 - Firefly [8]
Lieutenant Allison Jakes [4]
 - Firefly [8]
Storm Strider [18]
Storm Strider [18]
